tests/func/test_init.py

```diff
@@ -66,7 +66,7 @@ def test_cli(self):
         self._test_init()
 
 
-def test_init_quiet_should_not_display_welcome_screen(git, caplog):
+def test_init_quiet_should_not_display_welcome_screen(scm, caplog):
```

Comment:
Let's use `tmp_dir` fixture explicitly.

Reply:
@Suor Why though?

Reply:
scm creates a temp dir anyways, I see no good reason to put `tmp_dir` here. Esp since it is so obvious.

Reply:
1. We do it everywhere else.
2. It wasn't obvious for me, I needed to check the code to be sure it won't just `git init` in curdir.

Reply:
Added.